x/rollup/keeper/deposits.go

```diff
@@ -61,8 +62,8 @@ func (k *Keeper) processL1SystemDepositTx(ctx sdk.Context, txBytes []byte) (*der
 	return l1blockInfo, nil
 }
 
-// processL1UserDepositTxs processes the L1 user deposit txs and mints ETH to the user's cosmos address.
-func (k *Keeper) processL1UserDepositTxs(ctx sdk.Context, txs [][]byte) error { //nolint:gocritic // hugeParam
+// ProcessL1UserDepositTxs processes the L1 user deposit txs and mints ETH to the user's cosmos address.
+func (k *Keeper) ProcessL1UserDepositTxs(ctx sdk.Context, txs [][]byte) error { //nolint:gocritic // hugeParam
```

Comment:
This expands the API. Is there a reason to do this other than testing? Generally, it's better to test all behavior via the public interface (I think one of the only times we eschew that convention in our codebase is in the `comet` package for the paginate functionality).

Reply:
Nope, they were only exposed for test access. There were a bunch of modules in the Cosmos SDK that had similarly exposed funcs for testing purposes only, but I agree that it's not the best precedent to set.

Do you think it would be better to keep the separate `deposits_test.go` and `withdrawals_test.go` tests but move them into the `keeper` package to access the unexported funcs or delete them and try to test every error/edge case through `msg_server_test.go`?

Reply:
Any thoughts on testing all edge cases via the public interface `ApplyL1Txs`? I haven't investigated the tests in detail, so forgive me if I'm missing something obvious.

Reply:
Noting for visibility - we talked during our daily call and addressed this question, we'll move all unit tests into `msg_server_test.go` since we don't anticipate many more test cases being added